cmd/fs-v1.go

```diff
@@ -858,16 +885,18 @@ func (fs *FSObjects) getObjectETag(bucket, entry string, lock bool) (string, err
 		// Fetch the size of the underlying file.
 		fi, err = rlk.LockedFile.Stat()
 		if err != nil {
-			return "", toObjectErr(errors.Trace(err), bucket, entry)
+			logger.LogIf(ctx, err)
+			return "", toObjectErr(err, bucket, entry)
 		}
 
 		size = fi.Size()
 		reader = io.NewSectionReader(rlk.LockedFile, 0, fi.Size())
 	} else {
 		var err error
-		reader, size, err = fsOpenFile(fsMetaPath, 0)
+		reader, size, err = fsOpenFile(ctx, fsMetaPath, 0)
 		if err != nil {
-			return "", toObjectErr(errors.Trace(err), bucket, entry)
+			logger.LogIf(ctx, err)
```

Comment:
No need to log here. fsOpenFile already takes care of the err condition.

Reply:
done